src/controllers/reset.controller.js

```diff
@@ -0,0 +1,94 @@
+import { userService } from '../services/user.service.js';
+import { jwtService } from '../services/jwt.service.js';
+import { tokenService } from '../services/token.service.js';
+import { emailService } from '../services/emai.services.js';
+import { Token } from '../models/token.js';
+import { Op } from 'sequelize';
+import bcrypt from 'bcrypt';
+import { User } from '../models/user.js';
+import { ApiError } from '../exceptions/api.error.js';
+import { validatePassword } from '../utils/methods.js';
+
+const requestReset = async (req, res) => {
+  const { email } = req.body;
+
+  if (!email) {
+    throw ApiError.notFound('check your email');
```

Comment:
The error message 'check your email' might be misleading. It would be clearer to say something like 'Email is required' to indicate that the email field is missing.